apiary.apib

```diff
@@ -625,6 +625,54 @@ Get details for a single user.
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
 
+### users.invite [POST /users.invite]
+
+Invite a single user to your account
+
++ Request (application/json)
+
+    + Attributes (object)
+        + first_name: `john` (string, required)
+        + last_name: `doe` (string, required)
+        + email: `john@teamleader.eu` (string, required)
+
++ Response 204
+
+## Invites [/userInvites]
+
+*Required scopes: `users`*
+
+### invites.list [GET /userInvites.list]
```

Comment:
These titles should also be renamed right?

Reply:
Indeed, I also changed other titles/phrases.